============================= test session starts ==============================
platform darwin -- Python 3.7.4, pytest-5.0.1, py-1.8.0, pluggy-0.12.0 -- /Users/adiellevy/PycharmProjects/FrameWorkAutomation/venv/bin/python3.7
cachedir: .pytest_cache
metadata: {'Python': '3.7.4', 'Platform': 'Darwin-18.7.0-x86_64-i386-64bit', 'Packages': {'pytest': '5.0.1', 'py': '1.8.0', 'pluggy': '0.12.0'}, 'Plugins': {'html': '1.22.0', 'metadata': '1.8.0'}}
rootdir: /Users/adiellevy/PycharmProjects/FrameWorkAutomation
plugins: html-1.22.0, metadata-1.8.0
collecting ... collected 0 items / 1 errors

==================================== ERRORS ====================================
_________________ ERROR collecting Tests_f/home/login_test.py __________________
Tests_f/home/login_test.py:3: in <module>
    from Pages.home.login_page import LoginPage
Pages/home/login_page.py:2: in <module>
    from Base.selenium_driver import SeleniumDriver
Base/selenium_driver.py:20: in <module>
    class SeleniumDriver():
Base/selenium_driver.py:139: in SeleniumDriver
    self.log.info("*" * 70 + "\n" )
E   NameError: name 'self' is not defined
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
=========================== 1 error in 0.27 seconds ============================
